core/schema_utils.py

```diff
@@ -134,6 +136,15 @@ def normalize_against_schema(
                 prop[SCHEMA_KEY_SCHEMA],
                 global_validators=global_validators
             )
+    elif schema[SCHEMA_KEY_TYPE] == SCHEMA_TYPE_DICT_WITH_VARIABLE_NO_OF_KEYS:
```

Comment:
Thank you @chiragbaid7 for this -- can you elaborate more on its use case? 
For example name of the handler which requires this.

Reply:
There are instances where we have dictionarys the keys are not fixed, for example in `SkillMasteryDataHandler` the [mastery_change_per_skill](https://github.com/oppia/oppia/blob/0133c038a83193bfff270c2c45ee3406e78028ba/core/controllers/skill_mastery.py#L71) dict can have multiple skills ids of value type float.

```js
mastery_change_per_skill = {
           'skill_id_1': 0.9,
           'skill_id_2': 0.3,
           'skill_id_3': 0.1,
             ..
             ..
        }
```

Reply:
Got it. -- Thank you